### ISSUE TITLE
Why was the changelog removed from the repository and the documentation?

### ISSUE THREAD

**Issue by avilaton** [python-restx/flask-restx#193]
**Ask a question**
Why was the changelog removed from the repository and the documentation?

**Additional context**
Hi, I was looking at trying to move a project from flask_restplus==0.10.1 to restx but had a hard time figuring out what had changed. The docs say that is it mostly compatible with restplus but that is specific to a version, I think relabeling tags was a bad idea and can be reverted. Also having the changelog in the docs is not only very common but also a great way to figure out the risks when trying to move from restplus to restx.

**Comment by ziirish** [comment]
Some members from the maintainers team believe that CHANGELOG are hard to keep up-to-date.
The plan was to automate the *release notes* generation based on the commit messages but we didn't have time nor resources to actually implement this.

Anyway, you'll find we have added release notes to every release so far (for instance [here](https://github.com/python-restx/flask-restx/releases/tag/0.2.0))

**Comment by avilaton** [comment]
That made it so much harder for us to figure out what had changed and why the tag numbering had been restarted on flask-restplus. Having them in that part of github is a vendor lock-in and I think a bad idea for an open source project in general. They will not be in version control, moving to another place will mean having to migrate that separately.
I would also like to cite other examples in the community which have the changelog in the docs.
https://flask.palletsprojects.com/en/1.1.x/changelog/
https://marshmallow.readthedocs.io/en/stable/changelog.html
to name just a few.
Would you reconsider it please?

**Comment by ziirish** [comment]
I agree with you and I personally prefer the plain CHANGELOG file as well but this was a collective decision back then.

I'll ask the team again and get back to you when I have news.

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — looks shipped and closeable: the CHANGELOG is back.

Evidence:
- `CHANGELOG.rst` is at the repo root and is updated through to the current `1.3.3.dev` version, with entries grouped under each release.
- `CONTRIBUTING.rst:31` documents the changelog format and asks contributors to add an entry per PR (see top of `CHANGELOG.rst` for the prescribed structure).

If you were referring to the *online docs* (rather than the repo) and still see no changelog page in the ReadTheDocs build, point me at the URL and I'll dig further; otherwise would you mind closing this out?

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues on this repo; the references above were verified manually against current `master`.*
